src/routes/busRoutes.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import { addRoute, findAll, findOne, updateRoute, removeRoute, deleteAll } from '../controllers/RouteController.js';
+const router = Router();
```

Comment:
Can we move these routes in a folder called api?

Reply:
Done